ci-operator/step-registry/openshift/microshift/publish-release-notes/openshift-microshift-publish-release-notes-commands.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+set -xeuo pipefail
+
+if [[ "$JOB_NAME" == rehearse* ]]; then
+    echo "INFO: \$JOB_NAME starts with rehearse - running in DRY RUN mode"
+    export DRY_RUN=y
+fi
```

Comment:
I don't see `gen_ec_release_notes.py` making use of `DRY_RUN` env var

Reply:
I've updated the implementation here to pass the command line flag